app/Helpers/Storage.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Helpers;
+
+class Storage
```

Comment:
This class name is too generic, rename it to something more specific like User or move the code related to users specifically to a different class.